### ISSUE TITLE
lease default ttl of undefined causes infinite loop

### ISSUE THREAD

**Issue by mattzque** [mixer/etcd3#68]
Using v0.2.9, I'm using the same code as in the documentation to acquire a lease:

```
const lease = client.lease();
```

https://mixer.github.io/etcd3/classes/lease_.lease.html

But this causes the lease TTL to be undefined which causes an infinite loop for me. Code to reproduce:

```
const { Etcd3 } = require('etcd3');

const client = new Etcd3({ hosts: 'myhostname:2379' });
const lease = client.lease();
    
lease.on('lost', err => {
  console.log('etcd: lost', err);
});

lease.on('keepaliveFired', () => {
  console.log('etcd: keepaliveFired');
});

lease.on('keepaliveSucceeded', () => {
  console.log('etcd: keepaliveSucceeded');
});

lease.on('keepaliveFailed', () => {
  console.log('etcd: keepaliveFailed');
});

lease.on('keepaliveEstablished', () => {
  console.log('etcd: keepaliveEstablished');
});
```

It does work correctly if I give it a ttl like `client.lease(8)` does not cause an issue. But it took me a bit to figure this out, so I wanted to let you know. Maybe clarify documentation?

**Comment by willgm** [comment]
For some strange reason the same happens with TTL bigger than 6442450...

**Comment by connor4312** [comment]
Thanks for the report, this will be fixed in v0.2.10
